changelog/unreleased/issue-14086.toml

```diff
@@ -0,0 +1,15 @@
+type = "c"
+message = "Changed bulk indexing retry failure log-level from error to warning."
+
+issues = ["14086"]
+pulls = ["14088"]
+
+details.user = """
+The log-level for Opeansearch bulk indexing retry attempts has been changed from `ERROR` to `WARN`.
```

Comment:
Small typo in Opeansearch.

Should we still mention Elasticsearch as well? Otherwise it sounds as if we only applied this change in the case of OpenSearch.

Reply:
Thanks @boosty. I think mentioning both can't hurt. I'll open a small PR to fix the typo and mention both.